admin_guide/topics/deploying-cluster-auto-scaler.adoc

```diff
@@ -177,6 +177,12 @@ $ oc adm policy add-role-to-user cluster-autoscaler system:serviceaccount:cluste
 $ oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:cluster-autoscaler:cluster-autoscaler -n cluster-autoscaler
 ----
 
+. Add the following parameter to the cluster inventory file:
```

Comment:
@sunzhaohua2, is this what you meant in https://github.com/openshift/openshift-docs/pull/11924#issuecomment-422304897?